test/games/am2r/generator/test_am2r_base_patches.py

```diff
@@ -4,54 +4,94 @@
 import uuid
 from random import Random
 
+import pytest
+
+from randovania.game_description.db.dock import DockWeakness
 from randovania.games.am2r.generator.base_patches_factory import AM2RBasePatchesFactory
 from randovania.games.am2r.layout.am2r_configuration import AM2RConfiguration
 from randovania.layout import filtered_database
 
-_door_list = [
-    1047,
-    1751,
-    1259,
-    596,
-    1258,
-    1279,
-    1280,
-    1454,
-    1137,
-    878,
-    409,
-    1246,
-    1248,
-    1330,
-    1281,
-    1336,
-    1331,
-    1393,
-    1361,
-    1491,
-    1257,
-    1416,
-    1614,
-    1561,
-    1720,
-    1656,
-    601,
-]
-
-
-def test_base_patches(am2r_game_description, preset_manager) -> None:
+_save_door_mapping = {
```

Comment:
Is it planned to have other values here? "Normal Door" everywhere seems very static.

Reply:
no plans to have other values, its static. removed this mapping, and reverted it back to a list of indices